controller/lib/src/internal_api/interface.rs

```diff
@@ -15,7 +15,6 @@ impl InternalAPIInterface {
         );
 
         for _ in 1..num_workers {
-            let address = address.clone();
```

Comment:
Merge that change into the previous commit, please.

Reply:
Done !